unified-codes/data/v2/products.csv

```diff
@@ -1,1185 +1,1384 @@
+Cloxacillin,,,Parenteral,Powder for reconstitution,,1g,,,,,,,933f3f00,e0bb9d71,af1eb500,bf56a400,,3356fc00,,,,,,,2625,6.2.1,,10034041000116108,10326081000116106,51283501,,,,,,,,,,,,,
+Cloxacillin,,,Parenteral,Powder for reconstitution,,500mg,,,,,,,933f3f00,e0bb9d71,af1eb500,bf56a400,,3a9b74bf,,,,,,,2625,6.2.1,6.2.1,10034041000116108,10326881000116102,51283501,,,,,,,,,,,,,
+Clozapine,,,Oral,Tablet,,100mg,,,,,,,933f3f00,144fe724,33decb00,f4ee6200,,3a0db4bf,,,,,,,2626,24.1,24.1,10165681000116108,10165791000116100,51332703,,,,,,,,,,,,,
+Clozapine,,,Oral,Tablet,,200mg,,,,,,,933f3f00,144fe724,33decb00,f4ee6200,,3a2854bf,,,,,,,2626,24.1,24.1,10165681000116108,10166151000116101,51332703,,,,,,,,,,,,,
+Clozapine,,,Oral,Tablet,,25mg,,,,,,,933f3f00,144fe724,33decb00,f4ee6200,,3a0404bf,,,,,,,2626,24.1,24.1,10165681000116108,10165701000116105,51332703,,,,,,,,,,,,,
+Clozapine,,,Oral,Tablet,,50mg,,,,,,,933f3f00,144fe724,33decb00,f4ee6200,,3a08d4bf,,,,,,,2626,24.1,24.1,10165681000116108,10165951000116108,51332703,,,,,,,,,,,,,
+Coagulation Factor IX,,,Parenteral,Powder for reconstitution,,1000IU,,,,,,,933f3f00,413491ac,10db9d00,0fbfec00,,38aa24bf,,,,,,,221099,11.2.2,11.2.2,44019551000116105,44166691000116100,51131819,,,,,,,,,,,,,
+Coagulation Factor IX,,,Parenteral,Powder for reconstitution,,500IU,,,,,,,933f3f00,413491ac,10db9d00,0fbfec00,,38a3b4bf,,,,,,,221099,11.2.2,11.2.2,44019551000116105,44019611000116105,51131819,,,,,,,,,,,,,
+Coagulation Factor VIII,Factor VIII,,Parenteral,Powder for reconstitution,,500IU,,,,,,,933f3f00,f9259e6d,cb0eff00,612dc600,,422dc4bf,,,,,,,4257,11.2.2,11.2.2,44019551000116105,,51131821,,,,,,,,,,,,,
+Coal Tar,,,Topical,Solution,,5%,473mL,,,,,,933f3f00,8bd32da4,76ad7100,1e9a7000,,3d6b74bf,5e8ada00,,,,,,2635,13.4,13.4,10164391000116102,10168461000116108,30121501,,,,,,,,,,,,,
+Codeine,codeine phosphate,,Oral,Tablet,,30mg,,,,,,,933f3f00,d9d1d59b,1d8f7300,2c7c6a00,,388fb4bf,,,,,,,2670,2.2,2.2,10758441000116109,10033841000116103,51372102,,,,,,,,,,,,,
+Compound Sodium Lactate,"Hartmann's, Ringer's Lactate, Lactated Ringer's",,Parenteral,Solution,,,1000mL,,,,,,933f3f00,53ea14bf,54a61500,a4408400,,,6ded3700,,,,,,,26.2,,20039851000116102,10363941000116104,51172327,,,,,,,,,,,,,
+Compound Sodium Lactate,"Hartmann's, Ringer's Lactate, Lactated Ringer's",,Parenteral,Solution,,,250mL,,,,,,933f3f00,53ea14bf,54a61500,a4408400,,,d7731e00,,,,,,,26.2,,20039851000116102,,51172327,,,,,,,,,,,,,
+Compound Sodium Lactate,"Hartmann's, Ringer's Lactate, Lactated Ringer's",,Parenteral,Solution,,,500mL,,,,,,933f3f00,53ea14bf,54a61500,a4408400,,,773d6900,,,,,,,26.2,,20039851000116102,10363871000116102,51172327,,,,,,,,,,,,,
+Condom Female,,,,,,,1,,,,,,77fcbb00,22c41465e,,,,,bf4be518,,,,,,,22.1.4,,47036841000116101,,53131622,,,,,,,,,,,,,
+Condom Male,,,,,,Apple,,,,,,,77fcbb00,3b3444bf,,,,46d1043e,,,,,,,,22.1.4,,47036841000116101,,53131622,,,,,,,,,,,,,
+Condom Male,,,,,,Grape,,,,,,,77fcbb00,3b3444bf,,,,46d8143e,,,,,,,,22.1.4, ,4
```

Comment:
Hm, what's going on with all the lines of just `,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,`?

Reply:
Ok, failed again. Trying to comment on the trailing lines at the end of the file, but github keeps moving my comment up!

Reply:
Ugh that's a good question, I think a side effect of exporting the xlsm spreadsheet from excel. Can probably safely remove the empty lines, but I don't know why it always treats everything as new lines rather than changes, can do a bit of investigation to see for next time

Reply:
Removed trailing empty lines with 4ae34a0db - it did a weird thing again and deleted/re-added Zinc sulfate 🙀 , have tested and it's still loaded in.

Reply:
Did a bit of reading and it sounds like it's due to the way `csv` files are structured (has some table metadata that git doesn't know about). There is a git plugin we may be able to use https://github.com/paulfitz/daff - but not sure whether it's worth going down that track at this stage given this stuff is going to go away as soon as we can build a proper admin portal?